include/clap/ext/ambisonic.h

```diff
@@ -1,9 +1,11 @@
 #pragma once
 
-#include "../../plugin.h"
+#include "../plugin.h"
 
 // This extension can be used to specify the channel mapping used by the plugin.
-
+//
+// This extension ID contains `draft', yet it is stable.
+// See conventions/extension-id.md for more info.
```

Comment:
Curious why others got compat ids and this one didn't?

Reply:
This is an overlook!

Reply:
Oh cool! Well that's why we do reviews I guess :)